test/RelayHubGasCalculations.test.ts

```diff
@@ -350,82 +440,84 @@ contract('RelayHub gas calculations', function ([_, relayOwner, relayWorker, rel
             if (requestedFee !== 0 && messageLength !== 0 && process.env.MODE === 'coverage') return
             // 50k tests take more than 10 seconds to complete so will run once for sanity
             if (messageLength === 50000 && requestedFee !== 10) return
-            it(`should compensate relay with requested fee of ${requestedFee.toString()}% with ${messageLength.toString()} calldata size`, async function () {
-              const beforeBalances = await getBalances()
-              const pctRelayFee = requestedFee.toString()
-              const senderNonce = (await forwarderInstance.getNonce(senderAddress)).toString()
-              const encodedFunction = recipient.contract.methods.emitMessage('a'.repeat(messageLength)).encodeABI()
-              const baseRelayFee = '0'
-              const relayRequest: RelayRequest = {
-                request: {
-                  to: recipient.address,
-                  data: encodedFunction,
-                  from: senderAddress,
-                  nonce: senderNonce,
-                  value: '0',
-                  gas: gasLimit.toString()
-                },
-                relayData: {
-                  baseRelayFee,
-                  pctRelayFee,
-                  gasPrice: gasPrice.toString(),
-                  relayWorker,
+            it(`should compensate relay with requested fee of ${requestedFee.toString()}% with ${messageLength.toString()} calldata size`,
```

Comment:
why does this tests appear as entirely new?

Reply:
It doesn't, it's indentation